tests/tests/client-struct-unmount-race.sh

```diff
@@ -0,0 +1,66 @@
+#
+# Unmount Server and unmount a client as it's replaying to a remaining server
+#
+
+t_require_mounts 3
+majority_nr=$(t_majority_count)
+quorum_nr=$T_QUORUM
+
+cur_quorum=$quorum_nr
```

Comment:
cur_quorum isn't changed and is just quorum_nr, remove it and use quorum_nr anywhere it was used.